core/jobs_test.py

```diff
@@ -689,15 +690,6 @@ def test_raises_error_with_existing_mapper_param(self):
                 job_id, taskqueue_services.QUEUE_NAME_DEFAULT,
                 additional_job_params={'entity_kinds': ''})
 
-    def test_raises_error_with_invalid_param_name(self):
```

Comment:
It seems to me that you should keep a test of this form in the general jobs_test.py -- I know there's a similar one in UserQueryJobsOneOffTest but the param name stuff seems like general functionality that should be tested generally (perhaps with a sample/mock job).

Could you explain why you deleted this particular test? Or could we reinstate it?

Is there a reason you dropped it?

Reply:
There was no reason, I just thought since there was already a job that uses mapper params I can write the tests in that job. But yea its better to have it in jobs_test. Done!